core/src/main/java/io/aiven/klaw/model/EnvModel.java

```diff
@@ -18,8 +18,8 @@ public class EnvModel implements Serializable {
   private String id;
 
   @NotNull
-  @Size(min = 3, max = 7, message = "Please fill in a valid name")
-  @Pattern(message = "Invalid name", regexp = "^[a-zA-Z0-9_-]{3,}$")
+  @Size(min = 2, max = 10, message = "Please fill in a valid name")
+  @Pattern(message = "Invalid environment name !!", regexp = "^[a-zA-Z0-9_.-]{3,}$")
```

Comment:
Will there be any issue retrieving existing environments that people might be using from the DB that are just 2 chars long?